sdk/batch/batch/package.json

```diff
@@ -2,9 +2,8 @@
   "name": "@azure/batch",
   "author": "Microsoft Corporation",
   "description": "BatchServiceClient Library with typescript type definitions for node.js and browser.",
-  "version": "9.0.0",
+  "version": "9.1.0",
```

Comment:
Since this is technically a breaking change, we should do a major version update here even if are deprecating version 9.0.0

Reply:
Done.